pkg/actors/actors.go

```diff
@@ -527,8 +551,8 @@ func (a *actorsRuntime) drainRebalancedActors() {
 				// cancel any reminders
 				reminders := a.reminders[actorType]
 				for _, r := range reminders {
-					if r.ActorType == actorType && r.ActorID == actorID {
-						reminderKey := a.constructCompositeKey(actorKey, r.Name)
+					if r.reminder.ActorType == actorType && r.reminder.ActorID == actorID {
```

Comment:
nit: it's a little weird that a reminder variable has a `reminder` field on it. This doesn't require a change, just calling it out in case we can make this more apparent as to what `r` is as opposed to `r.reminder`.

Reply:
I wrapped the previous reminder struct within a new struct that contains some metadata to help with the new format.